catalog/transformer/catalog-transformer-xlsx/pom.xml

```diff
@@ -84,8 +75,7 @@
                     <instructions>
                         <Embed-Dependency>
                             catalog-core-api-impl,
-                            platform-util,
-                            security-rest-clientapi
+                            platform-util
```

Comment:
platform-util is a bundle so it shouldn't be embedded. I must have missed this when I removed it from all the poms or maybe it was added after I changed things.

Reply:
ok I'll remove it